pybamm/plotting/plot_summary_variables.py

```diff
@@ -0,0 +1,75 @@
+#
+# Method for plotting/comparing summary variables
+#
+import numpy as np
+
+
+def plot_summary_variables(
+    solutions, output_variables=None, labels=None, testing=False, **kwargs_fig
+):
+    """
+    Generate a plot showing/comparing the summary variables.
+
+    Parameters
+    ----------
+    solutions : (iter of) :class:`pybamm.Solution`
```

Comment:
can this be made to work if only a single solution is provided not in a list, e.g.
```python3
if isinstance(solutions, pybamm.Solution):
    solutions = [solutions]
```